src/tests/ftest/container/snapshot.py

```diff
@@ -54,33 +54,20 @@ def setUp(self):
         set up method
         """
         super(Snapshot, self).setUp()
-        # get parameters from yaml file, set default
-        createmode = self.params.get("mode",
-                                     '/run/poolparams/createmode/', 511)
-        createuid = os.geteuid()
-        creategid = os.getegid()
-        createsetid = self.params.get("setname",
-                                      '/run/poolparams/createset/')
-        createsize = self.params.get("size", '/run/poolparams/createsize/')
         self.log.info("==In setUp, self.context= %s", self.context)
 
-        try:
-
-            # initialize a python pool object then create the underlying
-            # daos storage
-            self.pool = DaosPool(self.context)
-            self.pool.create(createmode, createuid, creategid,
-                             createsize, createsetid, None)
-
-            # need a connection to the pool with rw permission
-            #    DAOS_PC_RO = int(1 << 0)
-            #    DAOS_PC_RW = int(1 << 1)
-            #    DAOS_PC_EX = int(1 << 2)
-            self.pool.connect(1 << 1)
+        # initialize a python pool object then create the underlying
+        # daos storage and connect to it
+        self.prepare_pool()
+        # need a connection to the pool with rw permission
```

Comment:
If you need to resubmit, I think this comment could be removed as it no longer applies.

Reply:
Done